docs/UserGuide.md

```diff
@@ -226,7 +231,7 @@ Examples:
 
 ### 3.2.4. Listing all tasks: `list`
 
-Displays a list of all tasks names in TR4CKER.
+Displays a list of all pending tasks names in TR4CKER.
```

Comment:
i think can be more user-centric, so `If you would like to view a list of tasks, ...` like this?